src/webapp/pages/landing/LandingPage.tsx

```diff
@@ -1,40 +1,299 @@
-import { Typography } from "@material-ui/core";
 import React from "react";
-import { useHistory } from "react-router-dom";
-import { Card, CardGrid } from "$/webapp/components/card-grid/CardGrid";
+import { GetApp as DownloadIcon, Print as PrintIcon } from "@material-ui/icons";
+import {
+    Box,
+    Button,
+    Checkbox,
+    CircularProgress,
+    FormControlLabel,
+    Paper,
+    useTheme,
+} from "@material-ui/core";
+import {
+    MultipleSelector,
+    MultipleSelectorProps,
+} from "$/webapp/components/multiple-selector/MultipleSelector";
 import { useAppContext } from "$/webapp/contexts/app-context";
+import { BasicDataSet } from "$/domain/entities/BasicDataSet";
+import { Locale } from "$/domain/entities/Locale";
+import _c from "$/domain/entities/generic/Collection";
 import i18n from "$/utils/i18n";
 
 export const LandingPage: React.FC = React.memo(() => {
-    const history = useHistory();
-    const { currentUser } = useAppContext();
-
-    const cards: Card[] = [
-        {
-            title: i18n.t("Section"),
-            key: "main",
-            children: [
-                {
-                    name: "John",
-                    description: "Entry point 1",
-                    listAction: () => history.push("/for/John"),
-                },
-                {
-                    name: "Mary",
-                    description: "Entry point 2",
-                    listAction: () => history.push("/for/Mary"),
-                },
-            ],
+    const theme = useTheme();
+    const { compositionRoot, currentUser } = useAppContext();
+
+    const [options, setOptions] = React.useState({
+        includeHeaders: true,
+        allLanguages: false,
+        allDatasets: false,
+    });
+
+    const handleChange = React.useCallback(
+        (event: React.ChangeEvent<HTMLInputElement>) => {
+            setOptions({ ...options, [event.target.name]: event.target.checked });
         },
-    ];
+        [options]
+    );
+
+    const dataSetSelectorProps = useDatasetSelector();
+
+    const exportToExcel = React.useCallback(() => {
+        compositionRoot.dataSets.export.execute([]).run(
+            _res => {
+                // console.log(res);
```

Comment:
Nothing to do on success? or is it a TODO?

Reply:
Was a TODO indeed